### PR TITLE
Remove unused gSavedAge

### PR DESCRIPTION
**Pull request by Harry-Hopkinson**
For https://github.com/OpenRCT2/OpenRCT2/issues/21193

### CODE REVIEW

**Comment by CookiePLMonster** [comment]
Looking at your diff... is this variable even used?

**Comment by Harry-Hopkinson** [comment]
I noticed that too, I don't think it is ever used...

**Review by tupaschoal — COMMENTED** [review]
Seems like it used to be written when exporting to S6, but that likely got deleted when we started exporting only park files. Does that make sense @IntelOrca ?

**Review by IntelOrca — APPROVED** [review]
I think I considered this variable redundant when we already have tick count. Probably originally used for the prompt where it asks if you want to save before exiting.

**Review by tupaschoal — COMMENTED** [review]
Well, if it's of no use anymore, can you please update to delete the variable instead @Harry-Hopkinson ?

File: src/openrct2/entity/EntityRegistry.cpp [hunk 1]
```diff
@@ -10,6 +10,7 @@
 #include "EntityRegistry.h"
 
 #include "../Game.h"
+#include "../GameState.h"
```
Comment (tupaschoal) [inline src/openrct2/entity/EntityRegistry.cpp:13]: This include is not needed anymore

Comment (ZehMatt) [reply src/openrct2/entity/EntityRegistry.cpp:13]: It will be needed eventually, this can stay I think.

Comment (Harry-Hopkinson) [reply src/openrct2/entity/EntityRegistry.cpp]: I looked through the file and I do not think that the GameState include would ever be used.

**Review by tupaschoal — APPROVED** [review]

**Pull request merged**